neuralprophet/forecaster.py

```diff
@@ -1295,7 +1322,14 @@ def crossvalidation_split_df(
                 del folds[i][1]["ID"]
         return folds
 
-    def double_crossvalidation_split_df(self, df, freq="auto", k=5, valid_pct=0.10, test_pct=0.10):
+    def double_crossvalidation_split_df(
+        self,
+        df: pd.DataFrame,
+        freq: str = "auto",
+        k: int = 5,
+        valid_pct: float = 0.10,
```

Comment:
from a consistency perspective, we can remove the last 0.

Reply:
not related this PR, but happy to do so :)